### ISSUE TITLE
Newsletter subscription option

### ISSUE THREAD

**Issue by seisvelas** [FlowCrypt/flowcrypt-docs#62]
From support email: https://mail.google.com/mail/u/5/#inbox/FMfcgxwJXBxCRqkkrGshnKMTvqDzVHxC

User suggests we offer a newsletter than users can subscribe to. From the email:

> One thing I would suggest is for you to build-in a Newsletter Subscription option which further binds your customers to your product. You can release your program updates, upgrades, up-sale your clients to go from your fabulous free service to premium paid service.  And you could give security tips and suggestions as well (we always need those).
> 
> Keep up the GREAT WORK!!

Personally, I would love this. This would be so fun. However, objectively I don't really think there's a need for it, and at worst it's just something to maintain.

We could certainly try it as an experiment. Or not, what do you think?

**Comment by tomholub** [comment]
Yes I wanted to do this for a while, but I knew I would not have the time to write stuff.

We have some infrastructure for this - signing up for waitlists. We could just add a waitlist named "newsletter".
Btw there is an issue with the front-end https://github.com/FlowCrypt/flowcrypt-web/issues/184 but the back-end part works well.
Then over time we could figure out how to send emails, likely using a 3rd party service in EU.

**Comment by seisvelas** [comment]
Okay, PR's are up for the waitlist. 

In the meantime I'll start looking into EU-based providers, and present what I find here.

**Comment by seisvelas** [comment]
After a bit of searching I've found some EU-based options:

## Email Labs

https://emaillabs.io/en/

I'm starting with these people because they're my favorite of the bunch. Here is the pricing:

![Screenshot from 2020-10-02 01-24-44](https://user-images.githubusercontent.com/44826516/94898059-1fa80f80-044e-11eb-9096-0d8d7e449e5d.png)

Not bad! They are hosted in Poland 🇵🇱 

## Teneo

https://esp.teneo.be/

Pricing:

![Screenshot from 2020-10-02 01-31-30](https://user-images.githubusercontent.com/44826516/94898584-08b5ed00-044f-11eb-8e70-da74bf24b8d8.png)

Teneo is based in Belguim.

---

I found other EU-based providers that seemed less legit, to the extent I didn't think it was worth even including them. Most of the big providers are US based, but not all! Let me know your thoughts and we'll move forward from there.

**Comment by tomholub** [comment]
I have just signed up for AWS (EU region) for transactional emails. They also have marketing email functionality.. I may just go with that. If it looks awkward to use, then I may go with Email Labs as you suggested.

**Comment by seisvelas** [comment]
>I have just signed up for AWS (EU region) for transactional emails. They also have marketing email functionality.. I may just go with that. 

I interpreted EU-based very strictly, so I only included companies based in Europe. Other factors we should consider:

* How soon do we want to start this (how many people on waitlist?)
* How frequently would we send this out (I lean towards infrequency on this, maybe even every other month)
* Planning content, structure, tone, etc
* And most of all, what is our goal with a newsletter? What do we hope to accomplish that would be an improvement over the current situation?

**Comment by tomholub** [comment]
Agree these are good questions. Agree about cca every other month. 

The goal is less clear. But some users want to hear from us, see what is new, etc, so at the very least we could cater to those.

Another category of users want to get security updates (often customers). This one is a much more immediate need, so maybe we could even start with that.

**Comment by seisvelas** [comment]
Hmm, how about this: I could write a 'sample' newsletter discussing new features, security features, and upcoming FlowCrypt stuff. I'll post it here, then you can give me feedback on what you like and dislike.

A rough draft might make it easier to think about how we want to structure this, kind of like the literary version of a proof of concept app or startup demo. 

Let me know what you think and I'll write something up.

**Comment by tomholub** [comment]
Sounds good - we can try it that way

**Comment by tomholub** [comment]
You could make some stats about vulnerabilities submitted / fixed per week?

**Comment by tomholub** [comment]
Or bounties paid out per month

**Comment by seisvelas** [comment]
Okay, how about this for the basic structure:

---

### Subject

FlowCrypt Newsletter #[issue number here] - [Subject line here]

### Body

[Intro with basic overview of FlowCrypt news, things we want to draw attention to, etc]

#### What's coming

[Here we'd discuss new features we're working on, non-security bugs we've fixed, etc]

#### Security

[New security features, FC security news, external audits, etc]

Here are the bug bounties that have been paid out this month:

* Bug 1
* Bug 2
* More bugs...

#### Outtro

[Wrap things up briefly, positive vibes]

---

Okay, so following that structure, here is a sample newsletter issue that you can critique:

---

### FlowCrypt Newsletter #0 - Introducing the FlowCrypt Newsletter!

This has been an exciting month for FlowCrypt! We've been working towards integration with keys.openpgp.org, an exciting and fast growing key server, after receiving numerous requests from users. Additionally, our bug bounty program has continued strong, with plenty of high quality reports coming in and receiving quick patches and payouts. So without further ado, let's jump in!

#### What's coming

FlowCrypt's **enterprise and on-prem services** have undergone intensive development. Soon, enterprise customers should be able to take advantage of the entire range of FlowCrypt features, without ever relying on any off-premise services. 

Additionally, we hope to soon rerelease our **iOS beta** - we've been very pleased with the popularity of the beta so far and appreciate the kind words that have come from users requesting a swift rerelease of the beta on TestFlight.

#### Security

Security developments have been exciting as always at FlowCrypt! We've been establishing more rigorous operational security policies for employees, and receiving all kinds of interesting bug reports. It's been a real surprise just how beneficial the bounty program has been - it's uncovered several bugs in codebases that went unnoticed in previous external audits. Just goes to show that a diversity of tactics is necessary for hardening our systems and protecting our customers.

After all, we're only as strong as our weakest link!

Here are bug bounties that were patched and paid out this month:

* Browser extension built in inbox allowed HTML rendering (but not XSS, thankfully!) in subject lines
* The page parameter of the browser extension could be used to load arbitrary web pages

Thanks to Mart Gil, Mr Dott, and Yash Ahmed for the excellent reports sent in.

#### Thanks for reading!

Feedback and questions pertaining to the contents of this newsletter can be sent by responding to this email. We're very excited about the many ways people are using and benefitting from cryptography, and are always happy to receive feedback!

- Alex @ FlowCrypt

**Comment by seisvelas** [comment]
I'm no Hemmingway but let me know your thoughts and I'm sure with some editing we can make all kinds of improvements :)

**Comment by seisvelas** [comment]
A few observations from myself - the security section over-dominates. That makes sense because I do security and I wrote it, so it's biased towards what I know. If we're to make a proper issue, I'll spend more time learning about what's being worked on that may not relate so much to security and therefore I otherwise wouldn't know about (new extension features, announcements and such). 

Also, maybe to add some lightness to the letter we could add a "FlowCrypt Lore" kind of section, where we talk about more arcane questions and things that users might want to know.

**What made me think about this?**  Well, consider this Reddit thread: [Flowcrypt - open source encryption for Gmail
](https://www.reddit.com/r/privacytoolsIO/comments/b0m1s0/flowcrypt_open_source_encryption_for_gmail/). One of the comments says this:

> https://flowcrypt.com/terms
> 
>> If you are located in the USA, or paid with a credit card, service is provided by: NVimport Corp, USA (reseller).
>>
>> All other users and customers are served by: FlowCrypt Limited, Hong Kong.
> 
> 
> 
> When I search for "NVimport Corp" the only things I find are about a furniture store ?! Anyway.... this seem to be an interesting product but since I'm not using any Google products (you can tell by my NVimport search results I'm sure) anymore since some years now I can say it's not for me.

I'm sure many curious users have looked into NVimport Corp before and wondered the same. I myself looked it up a bunch and found random tidbits and clues. I assume it's just the means by which FlowCrypt is incorporated in the US. But nevertheless, it's so mysterious.

My point is, I'm sure there are all kinds of interesting random stories and mysteries people get curious about. So maybe we could have a section for explanations and stories like that. On the other hand enterprise users probably wouldn't care at all.

Again, just let me know! I could also ask some of the other devs what things they are working on that might be worth mentioning.

Or possibly the whole thing as I wrote it is not useful and we should start from scratch :) There are only 30 people on the newsletter waitlist so there's no huge rush.

**Comment by tomholub** [comment]
Very sorry I'm delaying on this, I should be in a better state to have a look this coming fall - I hope!
